tests/std/tests/P0009R18_mdspan_layout_stride/test.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <array>
+#include <cassert>
+#include <concepts>
+#include <cstddef>
+#include <mdspan>
+#include <span>
+#include <type_traits>
+
+#include "test_mdspan_support.hpp"
+
+using namespace std;
+
+template <class IndexType, size_t... Extents, size_t... Indices>
+constexpr void do_check_members(const extents<IndexType, Extents...>& ext,
+    const array<IndexType, sizeof...(Extents)> strs, index_sequence<Indices...>) {

```

Comment:
Suggestion: I would usually expect `array` to be passed by const reference, not by const value, although this doesn't affect test correctness or lead to test instability. Occurs below.